Applications/opensim-cmd/opensim-cmd_viz.h

```diff
@@ -60,18 +57,6 @@ Description of subcommands:
          TimeSeriesTableQuaternion, then the data are visualized as body
          orientations using x-y-z triads, and the --model and --layout
          flags can be used.
-
-Description of options:
-  g, geometry  Search for geometry mesh files in this path.
-  m, model     If visualizing orientation data and --layout is 'model',
-               the orientation data will be visualized according to the default
-               pose of this model.
-  a, layout    If visualizing orientation data, this option specifies the layout
-               of x-y-z triads as either in a 'line' (the default), in a
-               'circle', or according to the default pose of a 'model'. If the
-               value is 'model', then the --model option must be provided.
-  r, rotate    If visualizing orientation data, this option specifies the space
-               fixed Euler angles to apply to data specified in radians.
```

Comment:
Could you leave this documentation in the file, commented out, so we can add it back later?

Reply:
Absolutely, good idea 💡